src/app/public/src/modules/analytics/google-analytics.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+import { Router, NavigationEnd } from '@angular/router';
+
+import { StacheWindowRef, StacheConfigService } from '../shared';
+
+@Component({
+  selector: 'stache-google-analytics',
+  template: ``
+})
+export class StacheGoogleAnalyticsComponent {
+  public constructor(
+    private router: Router,
+    private windowRef: StacheWindowRef,
+    private configService: StacheConfigService) {
+      if (this.configService.runtime.command === 'build') {
+        this.init();
+      }
+    }
+
+  public init(): void {
+    const clientId = this.configService.skyux.appSettings.stache.googleAnalyticsClientId;
+    this.windowRef.nativeWindow.eval(`
```

Comment:
Yep, if you're not careful :) Angular uses it everywhere. Basically, `eval` is appropriate when the content you are evaluating is either static, or "known". https://stackoverflow.com/questions/197769/when-is-javascripts-eval-not-evil